1-Draft/RFC00XX-Native-Command-Error-Handling.md

```diff
@@ -282,8 +282,8 @@ The reported error record object will be the new type: `NativeCommandException`
 ### Preference variable resolves error handling conflicts
 
 In cases where an existing script already handles non-zero native command errors, the preference
-variable `$PSNativeCommandErrorAction` may be set to `$false`. Error handling behavior of the
-PowerShell cmdlets is handled separately with `$ErrorActionPreference`.
+variable `$PSNativeCommandUseErrorActionPreference` may be set to `$false`. Error handling behavior
```

Comment:
Making this name longer seems like the wrong direction to go.  Also, if it ends in `ActionPreference` then I would expect it to be of type `[System.Management.Automation.ActionPreference]` instead of `[bool]`.

Reply:
I agree that the variable being boolean and the name ending with `Preference` sounds weird. `PSNativeCommandErrorActionEnabled`,  `PSEnableNativeCommandErrorAction` better describes the implicit type of the variable. Still they are long names, though.